Binary files wxorig/src/.DS_Store and wxnew/src/.DS_Store differ
diff -uNr wxorig/src/common/fs_arc.cpp wxnew/src/common/fs_arc.cpp
--- wxorig/src/common/fs_arc.cpp	2010-07-18 18:53:56.000000000 -0400
+++ wxnew/src/common/fs_arc.cpp	2010-08-16 21:13:12.000000000 -0400
@@ -358,7 +358,7 @@
         right = rightPart.GetFullPath(wxPATH_UNIX);
     }
 
-    if (right.GetChar(0) == wxT('/')) right = right.Mid(1);
+    if (right.Length() && right.GetChar(0) == wxT('/')) right = right.Mid(1);
 
     if (!m_cache)
         m_cache = new wxArchiveFSCache;
